### PR TITLE
Add ConsensusParameters methods for scheduler/epochtime

### PR DESCRIPTION
**Pull request by ptrus**
TODO:
- [x] add test case for latest heights fix

### CODE REVIEW

File: go/consensus/api/light.go [hunk 1]
```diff
@@ -42,8 +43,8 @@ type Parameters struct {
 	Height int64 `json:"height"`
 	// Meta contains the consensus backend specific consensus parameters.
 	Meta []byte `json:"meta"`
-
-	// TODO: Consider also including consensus/genesis.Parameters which are backend-agnostic.
+	// Consensus are the backend agnostic consensus parameters.
+	Consensus genesis.Parameters `json:"consensus"`
```
Comment (kostko) [inline go/consensus/api/light.go:47]:
```suggestion
	// Parameters are the backend agnostic consensus parameters.
	Parameters genesis.Parameters `json:"parameters"`
```
Also these should probably be placed above `Meta`.

File: go/consensus/tendermint/full/full.go [hunk 2]
```diff
@@ -981,24 +988,37 @@ func (t *fullService) GetLastRetainedVersion(ctx context.Context) (int64, error)
...
+	return tmHeight, nil
+}
+
+func (t *fullService) GetTendermintBlock(ctx context.Context, height int64) (*tmtypes.Block, error) {
+	if err := t.ensureStarted(ctx); err != nil {
+		return nil, err
+	}
+
+	tmHeight, err := t.heightToTendermintHeight(height)
+	if err != nil {
```
Comment (kostko) [inline go/consensus/tendermint/full/full.go:1015]: Change to a switch.

File: go/consensus/api/api.go [hunk 3]
```diff
@@ -110,6 +110,9 @@ type ClientBackend interface {
 	// GetEpoch returns the current epoch.
 	GetEpoch(ctx context.Context, height int64) (epochtime.EpochTime, error)
 
+	// EpochtimeConsensusParameters returns the epochtime consensus parameters.
```
Comment (kostko) [inline go/consensus/api/api.go:113]: Maybe file an issue to remove this when the new beacon backend is in place? Or mention it in #3180.

**Review by kostko — APPROVED** [review]

File: go/consensus/tendermint/full/full.go (hunk 2 above)
Comment (ptrus) [reply go/consensus/tendermint/full/full.go]: yeah good idea :grin: done

File: go/consensus/api/api.go (hunk 3 above)
Comment (ptrus) [reply go/consensus/api/api.go:114]: will comment on #3180 after i merge this

**Pull request merged**